gcovr/tests/test_gcovr.py

```diff
@@ -69,6 +81,12 @@ def assert_xml_equals(coverage, reference):
     compare_xml(reference_repn, coverage_repn, tolerance=1e-4, exact=True)
 
 
+def assert_json_equals(coverage, reference):
```

Comment:
I guess in this function you have content of two raw text files and it means you should compare coverage and reference as a text or use json.load(), and next compare python objects (I think you can use `compare_repn()` function for this from `pyutilib` like for `assert_xml_equals()`).

I used `json.dump()` with sort_keys=True during generation of json file to keep keys sorted and next compare coverage and reference as a two texts. Please check my Pull Request #326 because I had the same problems like you have :).

It is just my suggestion and @latk is an expert here :).

Reply:
Thanks @CezaryGapinski! I appreciate the help. I can't wait to get this merged in.